collect_app/src/main/java/org/odk/collect/android/preferences/FormManagementPreferences.java

```diff
@@ -91,44 +92,36 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
         if (key.equals(KEY_FORM_UPDATE_MODE) || key.equals(KEY_PERIODIC_FORM_UPDATES_CHECK)) {
             String newValue = sharedPreferences.getString(KEY_FORM_UPDATE_MODE, null);
             updateDisabledPrefs(newValue, sharedPreferences.getString(KEY_PROTOCOL, null));
-
-            Preference preference = findPreference(KEY_FORM_UPDATE_MODE);
-            preference.setSummary(((ListPreference) preference).getEntry());
         }
     }
 
     private void setupFormUpdateMode() {
         SharedPreferences sharedPreferences = preferencesProvider.getGeneralSharedPreferences();
         updateDisabledPrefs(sharedPreferences.getString(KEY_FORM_UPDATE_MODE, null), sharedPreferences.getString(KEY_PROTOCOL, null));
-
-        Preference formUpdateMode = findPreference(KEY_FORM_UPDATE_MODE);
-        formUpdateMode.setSummary(((ListPreference) formUpdateMode).getEntry());
     }
 
     private void updateDisabledPrefs(String formUpdateMode, String protocol) {
         Preference updateFrequency = findPreference(KEY_PERIODIC_FORM_UPDATES_CHECK);
         CheckBoxPreference automaticDownload = findPreference(KEY_AUTOMATIC_UPDATE);
 
         if (Protocol.parse(getActivity(), protocol) == Protocol.GOOGLE) {
-            findPreference(KEY_FORM_UPDATE_MODE).setEnabled(false);
-            automaticDownload.setEnabled(false);
+            displayDisabled(findPreference(KEY_FORM_UPDATE_MODE), getString(R.string.manually));
```

Comment:
To me it feels a lot simpler for us to simply display the disabled prefs with the appropriate values rather than change the underlying value. This has two advantages:

* The code is more declarative - we don't have to actually change values of other prefs when a pref changes (like changing the value of form update mode when the protocol changes).
* If the user switches between different options the disabled prefs will revert to the values they had them at before.